pkgs/applications/misc/octoprint/default.nix

```diff
@@ -34,6 +34,7 @@ let
           flask = super.flask.overridePythonAttrs (oldAttrs: rec {
             version = "2.2.5";
             format = "setuptools";
+            pyproject = null;
```

Comment:
Why not override `build-system` instead?

Reply:
No strong opinion. Was just focused on getting rid of the eval error.